mpas_analysis/sea_ice/modelvsobs.py

```diff
@@ -183,11 +193,11 @@ def seaice_modelvsobs(config, streamMap=None, variableMap=None):
         months = monthsInClim[climName]
         month = months[0]
         days = daysInMonth[month-1]
-        climatology = days*monthlyClimotology.sel(month=month)
+        climatology = days*monthlyClimatology.sel(month=month)
         totalDays = days
         for month in months[1:]:
             days = daysInMonth[month-1]
-            climatology += days*monthlyClimotology.sel(month=month)
+            climatology += days*monthlyClimatology.sel(month=month)
             totalDays += days
         climatology /= totalDays
```

Comment:
This should be converted to a function-- again, out of scope for this PR but worth noting for future refractors.